README.md

```diff
@@ -171,3 +171,11 @@ loaded it and passes the device tree to Salus.
 The above instructions use OpenSBI inbuilt in Qemu. If OpenSBI needs to be
 built from scratch, fw_dynamic should be used for `-bios` argument in the qemu
 commandline.
+
+### Vectors
+
+To run with the vector extension enabled requires a custom rust toolchain, due to a bug
+in nightly that prevents a no-std build with a custom target to not compile. Included in this
```

Comment:
Do you have a link to rust-lang bug that we can reference to figure out if/when this'll get fixed?

Reply:
will find and add it to the docs